examples/spring/src/main/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+graphql:
+  packages:
+    - "com.expediagroup.graphql.sample"
```

Comment:
this will be ignored because we override the config bean here -> https://github.com/ExpediaGroup/graphql-kotlin/blob/master/examples/spring/src/main/kotlin/com/expediagroup/graphql/sample/Application.kt#L49